src/reactA11yInputElementsRule.ts

```diff
@@ -45,12 +46,29 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 }
 
+function isExcludedInputType(node: ts.JsxSelfClosingElement): boolean {
+    let isExcludedType = false;
```

Comment:
Not: since this variable is only ever assigned to once, immediately before a return, let's remove it and directly return `true` or `false`. A little less code.